docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+
+All existing constraints for binary classification inherit from
+:code:`ClassificationMoment` and can be formulated as difference or ratio-based
+constraints.
+
+Difference-based constraints consider the difference with respect
+to a metric between a group of samples and the overall population or
+cohorts thereof.
+Parity represents the state where
+
+.. math::
+
+    \text{metric}_a - \text{metric}_* = 0 \; \forall a
+
+where :math:`a` is a group indentifier and :math:`*` represents the overall
+population. This is internally represented as two inequalities
+
+.. math::
+
+    \text{metric}_a - \text{metric}_* \leq 0 \; \forall a \\
+    -\text{metric}_a + \text{metric}_* \leq 0 \; \forall a
+
+The algorithms do not consider every kind of violation of these constraints,
+but rather violations that go beyond their :code:`difference_bound`. Such
+a difference-based constraint could be instantiated as
+
+    >>> ClassificationMoment(difference_bound=0.01)  # doctest: +SKIP
+
+Note that achieving parity with the :code:`difference_bound` does not mean
+that the difference between the group with the highest metric value and the
+group with the lowest metric value is below the :code:`difference_bound`, but
+rather the difference between each individual group and the metric value for
+the overall population (or cohort thereof) is below :code:`difference_bound`. 
+
+In constrast, ratio-based constraints consider the ratio as opposed to the
+difference. Parity is represented as
+
+.. math::
+
+    r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+with a :code:`ratio_bound` :math:`r`. Similar to the :code:`difference_bound`
+for difference-based constraints, ratio-based constraints have a
+:code:`ratio_bound_slack`. The two inequalities can also be considered
+separately as
+
+.. math::
+
+   r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \; \forall a \\
+   \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+and one step further in the equivalent form:
+
+.. math::
+
+   - \text{metric}_a + r \cdot \text{metric}_* \leq 0 \; \forall a \\
+   r \cdot \text{metric}_a - \text{metric}_* \leq 0 \; \forall a
+
+To use a constraint in its ratio-based version simply indicate as much through
+its arguments
+
+    >>> ClassificationMoment(ratio_bound=0.9, ratio_bound_slack=0.01)
+
+In other words, this means that
+:math:`0.9 \cdot \text{metric}_a \leq \text{metric}_*`, i.e., the metric value for
+each group needs to be at least 90% of the overall metric value, and
+:math:`0.9 \cdot \text{metric}_* \leq \text{metric}_a`, i.e., the overall metric
+value needs to be at least 90% of each group's metric value.
+
+.. note::
+
+    It is not possible to specify both :code:`difference_bound` *and*
+    :code:`ratio_bound` for the same constraint object.
+
+.. _demographic_parity:
+
+Demographic Parity
+^^^^^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *Demographic Parity* if
```

Comment:
A _binary_ classifier

Reply:
This is the binary classification section, but sure why not...

Reply:
Doesn't hurt to repeat.